v1/api/swagger/components/schemas/Profile.yaml

```diff
@@ -1,4 +1,13 @@
 type: object
+required: 
+  - TeamName
+  - TeamDesc
+  - Company
+  - Address
+  - Email
+  - EmailVerified
+  - PhoneNumber
+  - PhoneVerified
```

Comment:
As questions abouve, not sure what "required" means. These attributes are typically not required.

Reply:
See reply above.